### PR TITLE
[locale] pa-in.js correction 

### PR DESCRIPTION
**Pull request by adsingh14**
Hi!
I just re-corrected the pa(Punjabi) file, located in **locale**.

> Feel free to ask anything esp. about Punjabi.

### CODE REVIEW

**Comment by adsingh14** [comment]
Hi @marwahaha , how can I PR the locale folder '.js' file?

**Comment by marwahaha** [comment]
you should make changes to the src/locale/pa.js and src/test/locale/pa.js

**Comment by adsingh14** [comment]
@marwahaha, I had followed your given  instructions, but still 'Travis CI' checker failed. :(

**Comment by marwahaha** [comment]
You have some code style errors. You can review them here:
https://travis-ci.org/moment/moment/jobs/366232224

Thoughts @ksr89 @harpreetkhalsagtbit ?

**Comment by harpreetkhalsagtbit** [comment]
@marwahaha 
I have just looked at this issue. Found P.R of @adsingh14 . Is he able to resolve all the issues? I actually haven't got time to properly check this issue and his P.R. I will review it asap.

**Comment by adsingh14** [comment]
@harpreetkhalsagtbit Not yet, I've reverted the white-spaces. But still not getting the error. 😢 

You can check my previous repo:
https://github.com/moment/moment/pull/4548/commits/15378e8c19f9f189bf1aea37757d9b0382f30359

**Comment by adsingh14** [comment]
I'll upload the corrections later, maybe there is problem with _gruntfile.js_.

File: src/locale/pa-in.js [hunk 1]
```diff
@@ -1,41 +1,41 @@
 //! moment.js locale configuration
 //! locale : Punjabi (India) [pa-in]
-//! author : Harpreet Singh : https://github.com/harpreetkhalsagtbit
+//! author : Amandeep Singh : https://github.com/adsingh14
```
Comment (harpreetkhalsagtbit) [inline src/locale/pa-in.js]: @marwahaha 
I think this change should be reverted. As I am the original author of this file.

**Comment by harpreetkhalsagtbit** [comment]
@marwahaha 
I can confirm that Locale changes in this P.R are correct.

Will fix C.I issues

**Comment by adsingh14** [comment]
I'm going to re-upload it from a scratch.

**Comment by adsingh14** [comment]
I'm going to close this PR & re-open it.

**Comment by adsingh14** [comment]
@harpreetkhalsagtbit 
Re-uploaded version: https://github.com/moment/moment/pull/4567

All checks has passed.

**Pull request closed without merging**